markup.go

```diff
@@ -160,6 +162,35 @@ func Class(class ...string) Applyer {
 	})
 }
 
+// mustValidateClassNames ensures no class names have spaces
+// and panics with clear instructions on how to fix this user error.
+func mustValidateClassNames(class []string) {
+	for _, name := range class {
+		if strings.Contains(name, " ") {
+			failClassName(name)
+		}
+	}
+}
+
+// failClassName produces a helpful error message.
+// to fix the spaces in vecty.Class for example:
+// vecty.Class("hello there") will result in the following panic:
+// Spaces are not allowed in class names. Use `vecty.Class("hello", "there")` instead of `vecty.Class("hello there")`
+func failClassName(name string) {
```

Comment:
I don't think that we should take on the task of generating panics that instruct the user how to fix their code. I think just a panic saying:

```
panic: vecty: invalid class string with spaces passed to vecty.Class
```

Is enough to guide the user in the right direction. If needed, we can offer a help/documentation page for panic messages that goes into further depth about how to fix them.

Reply:
Agree, though I'd append the invalid class string to that message.